examples/todo/todo.py

```diff
@@ -1,12 +1,28 @@
 from flask import Flask
 from marshmallow import fields, pre_dump, Schema
 
-from flask_rebar import Rebar, errors, HeaderApiKeyAuthenticator
+from flask_rebar import Rebar, errors, HeaderApiKeyAuthenticator, Tag, \
+    SwaggerV2Generator
```

Comment:
Check out https://black.readthedocs.io/en/stable/the_black_code_style.html#line-length

Would this all fit on one line using an 88-char max line length?

Reply:
No.
Here's what black makes of the imports (with the default config):
`from flask import Flask
from marshmallow import fields, pre_dump

from flask_rebar import (
    Rebar,
    errors,
    HeaderApiKeyAuthenticator,
    Tag,
    SwaggerV2Generator,
)
from flask_rebar.validation import RequestSchema, ResponseSchema`

Reply:
note: I will not add this change here, since it is a style migration that is independent of original scope of 'add tags to the todo example'. I'm happy to put this into a new PR if that is what we decide (see #60 )